internal/pkg/cli/pipeline_status.go

```diff
@@ -60,7 +60,7 @@ func newPipelineStatusOpts(vars pipelineStatusVars) (*pipelineStatusOpts, error)
 		return nil, fmt.Errorf("new workspace client: %w", err)
 	}
 
-	session, err := sessions.NewProvider().Default()
+	session, err := sessions.NewProvider(sessions.UserAgentExtras("pipeline status")).Default()
```

Comment:
I guess we'd need to make sure then all sessions are using this provider.

Reply:
It's actually not so bad! because `NewProvider` is always a singleton 😄 

so if users call `sessions.NewProvider()` again afterwards it'll still fetch the same one that had "pipeline status" as a user agent 😎